workflow/cron/operator.go

```diff
@@ -34,6 +34,8 @@ type cronWfOperationCtx struct {
 	cronWfIf    typed.CronWorkflowInterface
 	log         *log.Entry
 	metrics     *metrics.Metrics
+	// scheduledTimeFunc returns the nearest scheduled time when it is called
```

Comment:
is this comment correct? is it the last scheduled time?

Reply:
Yes, the last scheduled time. Will correct